src/index.jsx

```diff
@@ -1,29 +1,17 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import {
-  ApolloClient, gql,
+  ApolloClient,
   InMemoryCache
 } from '@apollo/client'
 
 import { App } from 'app'
 
-const client = new ApolloClient({
-  uri: 'https://48p1r2roz4.sse.codesandbox.io',
+ApolloClient({
```

Comment:
We DO need the client in a constant, so that we can export it and use it throughout the application.

Reply:
Also, it should be located in a separate file under the `/startup` directory.

Reply:
Done